library/src/main/java/com/getstream/sdk/chat/view/MessageListView.java

```diff
@@ -160,50 +161,65 @@ public Drawable getDrawableForAttachment(Message message, Boolean mine, List<Mes
                     if (style.getMessageBubbleDrawableMine() != null)
                         return style.getMessageBubbleDrawableMine();
 
+                    topLeftRadius = style.getMessageTopLeftCornerRadiusMine();
+                    topRightRadius = style.getMessageTopRightCornerRadiusMine();
+                    bottomRightRadius = style.getMessageBottomRightCornerRadiusMine();
+                    bottomLeftRadius = style.getMessageBottomLeftCornerRadiusMine();
                     bgColor = style.getMessageBackgroundColorMine();
                     strokeColor = style.getMessageStrokeColorMine();
                     strokeWidth = style.getMessageStrokeWidthMine();
-                    if (!isDefaultBubble()){
-                        topLeftRadius = style.getMessageTopLeftCornerRadiusMine();
-                        topRightRadius = style.getMessageTopRightCornerRadiusMine();
-                        bottomRightRadius = style.getMessageBottomRightCornerRadiusMine();
-                        bottomLeftRadius = style.getMessageBottomLeftCornerRadiusMine();
-                    }else{
+                    if (isDefaultBubble()) {
+                        try {
+                            if (message.getAttachments() != null
```

Comment:
we do this often in our code. something like message.hasAttachments() would be nicer (checking both for null or zero size)

Reply:
Good idea.